kurtosis-devnet/isthmus.yaml

```diff
@@ -31,7 +31,7 @@ optimism_package:
         tolerations: []
         count: 1
       - el_type: op-reth
-        el_image: ""
+        el_image: "ghcr.io/paradigmxyz/op-reth:nightly@sha256:cc2fce901755783b42f5c76b8adc81375c7a80b8ca080b76b6e35cfb4d3e41a7"
```

Comment:
AFAIK this does not point to the op-reth that was fixed for isthmus

Reply:
Follow up PR worked at https://github.com/ethereum-optimism/optimism/pull/15122, ensuring to make CIs all pass